### PR TITLE
[CI:DOCS] Add support for `podman login --verbose`

### PR DESCRIPTION
**Pull request by flouthoc**
Reference Issue: https://github.com/containers/podman/issues/6717
Depends on PR:  https://github.com/containers/image/pull/1204 and https://github.com/containers/common/pull/607

**NOTE**: Following PR should be only merged when https://github.com/containers/image/pull/1204 & https://github.com/containers/common/pull/607 is merged in upstream and master is rebased.

Sample output
```bash
./podman login quay.io -v
Username: flouthoc
Password: 
Modified: /run/user/1000/containers/auth.json
Login Succeeded!
```

### CODE REVIEW

**Comment by rhatdan** [comment]
Why not just make this the default. No need for verbose.  We already print out a string.  Just add you content.  I don't think people are using the out put of podman login command in a scripts, they  should just be checking the exit code.

**Comment by flouthoc** [comment]
@rhatdan I think adding it as default text would require `SetCredentials` to return modified path as @mtrmac suggested or I can create a new function in https://github.com/containers/image if we are cool with just printing this as by default and call that in Podman. I'll try to see if we have any good approach instead of passing `verbose` flag.

**Review by TomSweeneyRedHat — COMMENTED** [review]
Needs adjustments on the man page to list the new option

**Comment by zhangguanzhang** [comment]
I'd prefer to add a test for it.
need sign the commit messages, could use `git commit --amend -s` and `git push -f`

**Comment by rhatdan** [comment]
@flouthoc @zhangguanzhang Any progress on this?

**Comment by flouthoc** [comment]
@rhatdan Sorry was not able to work on this for a while .Since this PR needs a major revamp and new API additions as per discussions here https://github.com/containers/podman/issues/6717 and https://github.com/containers/image/pull/1204. I'll put a proposal for those API changes then we can revisit this. But this PR will be revamped after that.

**Comment by flouthoc** [comment]
@rhatdan Updated this but following PR is dependent on updated https://github.com/containers/image/pull/1204 and https://github.com/containers/common/pull/607.

**Comment by zhangguanzhang** [comment]
need update the packages in your pr

**Comment by flouthoc** [comment]
@zhangguanzhang @TomSweeneyRedHat Sure i'll update packages, man page and tests once PR's in upstream dependencies are merged.

**Comment by flouthoc** [comment]
This PR is not needed anymore everything related to `verbose` is handled in -> https://github.com/containers/common/pull/607.

**Comment by flouthoc** [comment]
@rhatdan @TomSweeneyRedHat This feature already got added in https://github.com/containers/podman/pull/10690 and even docs got updated. Just adding example if necessary otherwise we can drop this PR.

File: docs/source/markdown/podman-login.1.md [hunk 1]
```diff
@@ -111,6 +111,14 @@ $ echo $testpassword | podman login -u testuser --password-stdin docker.io
 Login Succeeded!
 ```
 
+```
+$ podman login quay.io --verbose
+Username: flouthoc
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-login.1.md:116]:
```suggestion
Username: myusername
```

Comment (TomSweeneyRedHat) [reply docs/source/markdown/podman-login.1.md:116]: We try to keep examples like this generic, otherwise LGTM

Comment (flouthoc) [reply docs/source/markdown/podman-login.1.md]: @TomSweeneyRedHat Resolved.

**Comment by flouthoc** [comment]
@TomSweeneyRedHat Applied requested changes. Could you please review again ?

**Comment by TomSweeneyRedHat** [comment]
LGTM

File: docs/source/markdown/podman-login.1.md [hunk 2]
```diff
@@ -111,6 +111,14 @@ $ echo $testpassword | podman login -u testuser --password-stdin docker.io
 Login Succeeded!
 ```
 
+```
+$ podman login quay.io --verbose
+Username: myusername
+Password:
+Modified: /run/user/1000/containers/auth.json
```
Comment (rhatdan) [inline docs/source/markdown/podman-login.1.md:118]: Doesn't this like show up as

credential helper: /run/user/1000/containers/auth.json

Comment (rhatdan) [reply docs/source/markdown/podman-login.1.md:118]: Here is what I got.

Used:  /run/user/3267/containers/auth.json

Comment (flouthoc) [reply docs/source/markdown/podman-login.1.md]: @rhatdan Fixed.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan, PTAL and merge if it LGTY

**Pull request merged**